doc/_static/diagrams/sphinx_build_flow.dot

```diff
@@ -13,26 +13,54 @@ digraph build {
         shape=record
         label = "Sphinx | <init> __init__ | <build> build"
     ];
+    "legend" [
+        shape=record
+        label = <<table border="0" cellborder="0" cellspacing="0">
+            <tr><td align="center"><u><b>Method types</b></u></td></tr>
+            <tr><td align="left"><font color="red">Final</font></td></tr>
+            <tr><td align="left"><font color="darkorange">Overridable</font></td></tr>
+            <tr><td align="left"><font color="darkgreen">Abstract</font></td></tr>
```

Comment:
IIRC, green and red aren't colorblind friendly so I'd suggest either changing one of them or changing the font-family of the text (e.g., italic for the abstract methods).

Reply:
done